ds000117/sub-01/ses-meg/meg/sub-01_ses-meg_task-facerecognition_run-01_events.tsv

```diff
@@ -1,147 +1,147 @@
 onset	duration	onset_sample	stim_type	trigger	stim_file
-26.078	0	26628	Unfamiliar	13	meg/u032.bmp
```

Comment:
why is duration = 0? In the publication, it says: "the critical stimulus (face or scrambled face) was superimposed for a random duration between 800 and 1,000 ms." 

And in the BIDS specification, it says: "A 'duration' value of zero implies that the delta function or event is so short as to be effectively modeled as an impulse."